javaparser-core/src/main/java/com/github/javaparser/ast/type/ClassOrInterfaceType.java

```diff
@@ -29,7 +30,7 @@
 /**
  * @author Julio Vilmar Gesser
  */
-public final class ClassOrInterfaceType extends Type {
```

Comment:
that is not the name of the node, that is a "reference by name"... I think this shouldn't be a `NamedNode`